docs/management/admin/host-isolation-ov.asciidoc

```diff
@@ -69,6 +74,21 @@ NOTE: The response console is an https://www.elastic.co/pricing[Enterprise subsc
 . Press *Return*.
 ====
 
+.Automatically isolate a host through a rule's endpoint response action
```

Comment:
```suggestion
.Automatically isolate a host using a rule's endpoint response action
```

Reply:
I went back and forth on this, so I'm glad you left this suggestion :)